app/views/shared/_upload_form.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= form_for @file do |f| %>
-   <%= f.file_field :attachment %>
-   <%= f.text_field :folder_id, value: "#{current_folder.id}", type: :hidden %>
-   <%= f.submit "Upload File" %>
- <% end %>
+  <%= f.file_field :attachment %>
+  <%= f.text_field :folder_id, value: "#{current_folder.id}", type: :hidden %>
```

Comment:
We may be able to refactor and remove these from our forms entirely now that current_folder is tracked in the session.